v1/transactions/transactions.gunk

```diff
@@ -124,17 +124,33 @@ type GetTransactionRequest struct {
 
 // GetTransactionsRequest is the request envelope to get a list of transactions.
 type GetTransactionsRequest struct {
+	// FromDate is starting date to get the transaction
+	FramDate string `pb:"1" json:"from _date"`
```

Comment:
Can we use timestamp for this instead of string?

Reply:
I think i will still use string in this commit, because another Date field in this Repo use string as data type.

Reply:
That's wrong - gunk supports `time.Time` as a protobuf timestamp, so please change both.

Reply:
well noted.
will change all the Date field from string to time.Time

Reply:
updated, PTAL.